src/java/org/apache/cassandra/db/commitlog/CommitLogDescriptor.java

```diff
@@ -58,19 +58,19 @@
     static final String COMPRESSION_CLASS_KEY = "compressionClass";
 
     // We don't support anything pre-3.0
-    public static final int VERSION_30 = 6;
-    public static final int VERSION_40 = 7;
+    private static final int COMMITLOG_VERSION_30 = 6;
```

Comment:
for some reason C* abandoned type safety for serialization versions and uses plain `int`. Still, commit log version is not the same thing as `MessagingService` versions...

Reply:
maybe rename back since those fields are private now, less changes the better

Reply:
ok, I rename back and add a comment